config/routes.rb

```diff
@@ -197,6 +197,12 @@ def draw(routes_name)
     get 'projects/sidebar', to: 'projects#sidebar', as: 'projects_sidebar'
     get 'projects/dt_state_load', to: 'projects#dt_state_load', as: 'projects_dt_state_load'
 
+    resources :project_folders, only: [] do
```

Comment:
I think you have to rebase to the current feature branch and merge with existing routes?

Reply:
Or is this route on different level? 🤔

Reply:
It's the same level, I will rebase